rpcs3/ds4_pad_handler.cpp

```diff
@@ -394,6 +398,21 @@ void ds4_pad_handler::ProcessData()
 			pad.m_buttons[12 + i - 4].m_value = pressed ? 255 : 0;
 		}
 
+#ifdef _WIN32
+		bool any_button_pressed = false;
+
+		for (int i = 6; i < 16; i++)
+		{
+			if (pad.m_buttons[i].m_pressed)
+			{
+				any_button_pressed = true;
```

Comment:
Could you move the SetThreadExecutionState command here and remove need for the any_button_pressed variable?